doc/developer/debugging.rst

```diff
@@ -85,7 +85,7 @@ Edit a file in a running apache WSGI container
 
    docker-compose exec geoportal bash
    vi ...
-   kill -s USR1 1  # graceful
+   kill -s HUP `ps aux|grep gunicorn|head --lines=1|awk '{print $2}'`  # graceful
```

Comment:
Then, the title is wrong...

Why don't you suggest to put gunicorn in devel mode (the --reload option) instead?